meinberlin/assets/scss/style_dashboard.scss

```diff
@@ -96,8 +97,8 @@
 // @import "components/typeahead";
 @import "components/upload";
 
-@import "print";
-@import "utility";
-@import "shame";
+@import "styles_dashboard/print";
+@import "styles_dashboard/utility";
+@import "styles_dashboard/shame";
```

Comment:
moved component that definitely belong to the dashboard over and leaving rest for now, will be easier to move/delete once more user pages are changed